src/main/cpp/audioengine.h

```diff
@@ -20,109 +20,122 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
-#ifndef __AUDIOENGINE_H_INCLUDED__
-#define __AUDIOENGINE_H_INCLUDED__
+#ifndef __MWENGINE__AUDIOENGINE_H_INCLUDED__
+#define __MWENGINE__AUDIOENGINE_H_INCLUDED__
 
 #include "audiobuffer.h"
 #include "audiochannel.h"
 #include "global.h"
 #include "processingchain.h"
 
-/**
- * this is the API exposed to an external application
- * for interacting with the AudioEngine (it does nothing
- * more than prepare and start / stop / reset the render thread
- */
-namespace AudioEngine
+namespace MWEngine {
+class AudioEngine
 {
-    /* public methods */
+    /**
```

Comment:
Changing AudioEngine from a namespace to a static class. This also makes eligible for exposing it through SWIG to Java. Cleaning up rudimentary methods (defined in _javabridge_api.h_) and including AudioEngine in the SWIG interface file.